src/main/resources/static/ptmexchange/index.html

```
@@ -25,16 +25,16 @@ <h1>Overview</h1>
     <h1>Datasets</h1>
     <table class="pcdatatable" style="border-collapse:collapse;">
       <tr>
-	<th>Species</th><th>Name</th><th>Modification</th><th>PXD</th><th>PeptideAtlas</th><th>UniProtKB</th><th>Publication</th><th>Summary Results</th><th>Search Database</th>
+	<th>Species</th><th>Name</th><th>Modification</th><th>PXD</th><th>PeptideAtlas</th><th>UniProtKB*</th><th>Publication</th><th>Summary Results</th><th>Search Database</th>
       </tr>
       <tr class="pc-hov">
-        <td><i>Oryza sativa</i></td><td>Rice</td><td>Phospho</td><td><a href="http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD046188">PXD046188</a></td><td><a href="https://peptideatlas.org/builds/rice/phospho/">Rice Phospho 2022&#8209;11</td><td>Visible on <a href="https://www.uniprot.org/proteomes/UP000059680">Oryza sativa subsp. japonica (Rice) proteome</a></td><td><a href="https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00187">Ramsbottom et al., 2024</a></td><td><a href="./Files/Rice_GSB_phospho_all_prots_0924.csv.gz">  Results</a></td><td><a href="./Files/Osativa_super_annotation_union_noIC4R_v2_cRAP.fasta.gz"> Fasta file</a></td>
+        <td><i>Oryza sativa</i></td><td>Rice</td><td>Phospho</td><td><a href="http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD046188">PXD046188</a></td><td><a href="https://peptideatlas.org/builds/rice/phospho/">Rice Phospho 2022&#8209;04</td><td>Visible on <a href="https://www.uniprot.org/proteomes/UP000059680">Oryza sativa subsp. japonica (Rice) proteome</a></td><td><a href="https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00187">Ramsbottom et al., 2024</a></td><td><a href="./Files/Rice_GSB_phospho_all_prots_0924.csv.gz">  Results</a></td><td><a href="./Files/Osativa_super_annotation_union_noIC4R_v2_cRAP.fasta.gz"> Fasta file</a></td>
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </a> ], start tag match failed [ <a href="https://peptideatlas.org/builds/rice/phospho/"> ] on line 31.

(tag-pair)